csvqb/csvqb/models/cube/qb/catalog.py

```diff
@@ -32,7 +33,7 @@ def get_description(self) -> Optional[str]:
         return self.description
 
     def get_identifier(self) -> str:
-        return self.title
+        return self.identifier or self.title
```

Comment:
why we need an identifier?

Reply:
So the identifier just allows us to differentiate between datasets on the SPARQL end-point without having to use the title. And it gives the user an extra degree of flexibility to specify an identifier that's better/more specific than the title.